### DISCUSSION TITLE
Voice features.

### DISCUSSION THREAD

**Discussion by PoofImaFox** [cinderblocks/libremetaverse#98] [Q&A] [answered]
I've been able to figure out how to use most of this from looking at the unit tests.
But I'm not sure I understand how the voice works. Is there any built in way to stream data to the voice channel in the game from sound files like an mp3? Should I look into building this feature in a fork? Ideally this is useful for AI assistants. @cinderblocks Thanks for the help.

**Comment by cinderblocks** [comment, marked as answer]
I don’t think it’s feasible to do with Vivox, but I’m writing a client for the WebRTC replacement and that is something that could be done there.

**Reply by PoofImaFox** [reply]
Alright that sounds nice thank you. I see your WebRTC feature branch here.
